README.md

```diff
@@ -32,13 +32,13 @@ Custom Fields and Instance Tracking require TPP 18.2 or higher, and Token Authen
 3. Download the source code:
 
 ```sh
-go get github.com/Venafi/vcert
+go get github.com/Venafi/vcert/v4
 ```
 
 or
 
 ```sh
-git clone https://github.com/Venafi/vcert.git $GOPATH/src/github.com/Venafi/vcert
+git clone https://github.com/Venafi/vcert/v4.git $GOPATH/src/github.com/Venafi/vcert/v4
```

Comment:
the git url is invalid, the repo is still https://github.com/Venafi/vcert.git

Reply:
These have been updated and added the note from https://blog.golang.org/using-go-modules about being outside of `$GOPATH/src` when developing in module mode. Wondering if it worth adding the note for `replace github.com/Venafi/vcert => ../path` note for developing go mod locally in other repos when working with modules in go 1.13

Reply:
I don't think it is needed, this is standard practices by now.